static/js/sefaria/sefaria.js

```diff
@@ -1943,6 +1943,17 @@ _media: {},
           data["completions"][0].toLowerCase().replace('״','"') == query.slice(0, data["completions"][0].length).toLowerCase().replace('״','"') &&
           data["completions"][0] != query.slice(0, data["completions"][0].length))
   },
+  isGershayimVariant: function(query, data) {
+    function replace_gershayim_by_merkhaot(string) {
```

Comment:
I don't understand this function name. Should it be `replace_quotes_with_gershayim()`? (also, note the order of the replace will need to change if we switch to using unidecode).

Reply:
the order doesn't matter. we're not changing the server response. just how it compares query to data (i'll change the name)